src/pages/Admin/funkis/index.jsx

```diff
@@ -427,67 +237,32 @@ const FunkisAdminComponent = ({
-                <DataTableHeadCell>Bokade pass</DataTableHeadCell>
-              </DataTableRow>
-            </DataTableHead>
-            <DataTableBody>
-              {funkisar !== {} && Object.values(funkisar).sort((f, s) => {
-                const first = sortation.field === 'selectedFunkisAlt' && f.selectedFunkisAlt? positions[f[sortation.field]] : f[sortation.field]
-                const second = sortation.field === 'selectedFunkisAlt' && s.selectedFunkisAlt? positions[s[sortation.field]] : s[sortation.field]
-                if(first > second) return -1;
-                if(first < second) return 1;
-                return 0;
-              }).sort(() => sortation.dir).filter(f => {
-                for(const key of ['name', 'email', 'liuid']) {
-                  if(f[key] && f[key].toLowerCase().includes(searchTerm)) return true;
-                  console.log(f.selectedTimeSlots);
-                  const options = {day: 'numeric', month: 'numeric', hour:'numeric', minute:'numeric'};
-                  if(f.selectedTimeSlots) {
-                    for(const t of f.selectedTimeSlots) {
-                      const start = new Intl.DateTimeFormat('sv', options).format(idTimeslots[t].start_time)
-                      const end = new Intl.DateTimeFormat('sv', options).format(idTimeslots[t].end_time)
-                      console.log(start);
-                      console.log(end);
-                      if(start.replace(/ /g,'').includes(searchTerm.replace(/ /g,'')) || end.replace(/ /g,'').includes(searchTerm.replace(/ /g,''))) return true;
-                    }
-                  }
-                }
-                return false;
-              }).map((f) => {
-                    return (
-                      <FunkisAdminRow
-                        funkis={{
-                          ...f,
-                          selectedFunkisAlt: positions[f.selectedFunkisAlt],
-                          selectedTimeSlots: f.selectedTimeSlots.map(t => {
-                            const options = {day: 'numeric', month: 'numeric', hour:'numeric', minute:'numeric'};
-                            const start = new Intl.DateTimeFormat('sv', options).format(idTimeslots[t].start_time);
-                            const end = new Intl.DateTimeFormat('sv', options).format(idTimeslots[t].end_time);
-                            return (`${start} -  ${end}` );
-                          })
-                        }}
-                        onClick={() => {
-                          setActiveFunkis({
-                            ...f,
-                          });
-                          setFunkisModalOpen(true);
-                          setOriginalTimeslots(f.selectedTimeSlots);
-                        }}
-                      />
-                    );
-                  }
-              )}
-            </DataTableBody>
-          </DataTableContent>  
-        </DataTable>  
+        <Select 
```

Comment:
För enhetlighetens skull hade det varit bra med padding mellan Funkistyp och Klar. Se Checka in funkis.